api/v1alpha1/job.go

```diff
@@ -47,6 +47,12 @@ const (
 	Status       PowerAction = "status"
 )
 
+// Pointer provides an easy way to retrieve the power action as a pointer for use in job
```

Comment:
`&` is `an easy way to retrieve the power action as a pointer`. :-)

Other than 9 characters in each of the tests below, what does this add?

Is a `Pointer()` conventional or fulfill some interface?

Reply:
You can't retrieve a pointer to a constant so you need to create a variable every time you want to set this. This is a convenient thing and I'm debating making it Ptr()